src/chatgpt.rs

```diff
@@ -210,3 +221,71 @@ impl GptClient {
         result
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
♻️ Since these tests are for the config manager, they should probably live in config_manager.rs now that it exists.